src/components/Post/ExpandedPost.tsx

```diff
@@ -61,6 +61,7 @@ export default function ExpandedPost(
           address={user?.address}
           incognito={post.creator === ''}
           group={zkGroup}
+          twitterUsername={post.type === MessageType._TWEET ? post.creator : undefined}
```

Comment:
Can we rename the `twitterUsername` prop in `<Avatar />` to `username`?
As we integrate with further providers, it would be good for the components to be provider agnostic.

Reply:
will do - create #72 to track, will make the change when i work on UI for github and reddit onboarding